packages/neos-ui/src/Sagas/CR/NodeOperations/moveDroppedNode.js

```diff
@@ -1,39 +1,21 @@
 import {takeEvery} from 'redux-saga';
-import {put, select, call} from 'redux-saga/effects';
+import {put} from 'redux-saga/effects';
 
-import {selectors, actions, actionTypes} from '@neos-project/neos-ui-redux-store';
+import {actions, actionTypes} from '@neos-project/neos-ui-redux-store';
 
-import determineInsertMode from './determineInsertMode';
 import {calculateChangeTypeFromMode, calculateDomAddressesFromMode} from './helpers';
 
-export default function * moveDroppedNode({globalRegistry}) {
-    const nodeTypesRegistry = globalRegistry.get('@neos-project/neos-ui-contentrepository');
-    const canBeInsertedAlongsideSelector = selectors.CR.Nodes.makeCanBeInsertedAlongsideSelector(nodeTypesRegistry);
-    const canBeInsertedIntoSelector = selectors.CR.Nodes.makeCanBeInsertedIntoSelector(nodeTypesRegistry);
-
+export default function * moveDroppedNode() {
     yield * takeEvery(actionTypes.CR.Nodes.MOVE, function * handleNodeMove({payload}) {
-        const {nodeToBeMoved: subject, targetNode: reference} = payload;
-        const canBeInsertedAlongside = yield select(canBeInsertedAlongsideSelector, {subject, reference});
-        const canBeInsertedInto = yield select(canBeInsertedIntoSelector, {subject, reference});
+        const {nodeToBeMoved: subject, targetNode: reference, position} = payload;
 
-        const mode = yield call(
-            determineInsertMode,
```

Comment:
did we remove everything correctly related to the old behavior? IMHO "determineInsertMode" is not used anymore, is it? :) And the popup displaying the position selector can be removed as well, right?

Reply:
@skurfuerst it is used when (copy|cut)/pasting via the tree toolbar buttons

Reply:
https://github.com/neos/neos-ui/blob/32f82d6a9bf580d52a234cbc84f283b9207b745b/packages/neos-ui/src/Sagas/CR/NodeOperations/cutAndPasteNode.js#L39

Reply:
makes sense, thx <3